pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1096,7 +1096,7 @@ const (
 	// TiDBTTLRunningTasks limits the count of running ttl tasks. Default to 0, means 3 times the count of TiKV (or no
 	// limitation, if the storage is not TiKV).
 	TiDBTTLRunningTasks = "tidb_ttl_running_tasks"
-	// TiDBEnableVectorType indicates whether to enable VECTOR data type.
+	// TiDBEnableVectorType has been Deprecated. Not used anymore.
 	TiDBEnableVectorType = "tidb_enable_vector_type"
```

Comment:
Please just remove this variable. It has not been introduced to TiDB, so there is no "deprecate".